README.md

```diff
@@ -1,6 +1,38 @@
 # dw-proxy
-A reverse proxy based on the NGINX Reverse Proxy <br>
-- https://www.nginx.com/resources/admin-guide/reverse-proxy/ <br>
-- https://hub.docker.com/r/jwilder/nginx-proxy/ <br>
-if you are using SSL, put your shared.crt and shared.key in the 'certs'-directory
+
+A reverse proxy based on `nginx`, capable of automatically fetching, installing and renewing `letsencrypt`SSL certificates at no cost. So using it saves a lot of manual maintenance steps.
+
+# Usage
+
+Firstly, `docker` and `docker-compose` are required.
+
+Run only one instance of this component on the server to get a reverse proxy with SSL support in front of other services.
+
+The other services then only need to add three environment variables which will route traffic to them. In the `docker-compose.yml` file of the service, add something along these lines to expose it under a specific name with SSL support provided from `dw-proxy`:
+
+		environment:
+		  - VIRTUAL_HOST=beta-sso.dina-web.net
+		  - LETSENCRYPT_EMAIL=dina@mail.dina-web.net
```

Comment:
Nope, that is for the other one. Also the LETSENCRYPT_EMAIL is optional and will default to info@VIRTUAL_HOST if not set, see https://hub.docker.com/r/eforce21/letsencrypt-nginx-proxy/